scripts/linters/css_linter.py

```diff
@@ -147,9 +147,10 @@ def get_linters(config_path, files_to_lint, verbose_mode_enabled=False):
         verbose_mode_enabled: bool. True if verbose mode is enabled.
 
     Returns:
-        tlinter: object(linter). Returns linter object.
+        linter: tuple(ThirdPartyCSSLintChecksManager). Returns custom and
```

Comment:
Write the typeinfo as "(None, ThirdPartyCSSLintChecksManager)."

In the description, write "Returns a 2-tuple of ..." Ditto elsewhere.

Reply:
Done.